templates/globals/updatable_profile_image_popover_content..html

```diff
@@ -0,0 +1,13 @@
+{% load static %}
+
+<a class='btn m-0 p-0' role='button' style='width:100%'>
+    <label for='profileImage'> <span class='text-muted'>Change Profile Picture</span></label>
+    <input name='profile_image' type='file' id='profileImage' style='display:none' onchange='imageValidation()'>
+</a>
+<a href={% if profile.profile_image %}'{{ profile.profile_image.url }}'
```

Comment:
Check the file name.

Also, use `profile_image_url` here in `href` arrtibute.

Reply:
Also, add some CSS to both anchor tags to show cursor over them as `pointer` and add some hovering effect (background of the button should darken upon hover.